src/request/import/ImportWords.js

```diff
@@ -30,8 +30,9 @@ class ImportWords {
 
         /** @type {{entropy: Nimiq.Entropy?, privateKey: Nimiq.PrivateKey?}} */
         this._secrets = { entropy: null, privateKey: null };
-        /** @type {KeyguardRequest.KeyResult[]} */
-        this._keys = [];
+        // eslint-disable-next-line max-len
+        /** @type {{entropy: Partial<KeyguardRequest.SingleKeyResult>, privateKey: Partial<KeyguardRequest.SingleKeyResult>}} */
+        this._keys = { entropy: {}, privateKey: {} };
```

Comment:
The variable `this._keys` is misnamed, because it does not contain `Key`s (the class), but only (Single)`KeyResult`s (the object).